tricycle_controller/include/tricycle_controller/tricycle_controller.hpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2022 Pixel Robotics.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+ * Author: Tony Najjar
+ * Author: Sara Al Arab
+ */
+
+#ifndef TRICYCLE_CONTROLLER__TRICYCLE_CONTROLLER_HPP_
+#define TRICYCLE_CONTROLLER__TRICYCLE_CONTROLLER_HPP_
+
+#include <chrono>
+#include <cmath>
+#include <memory>
+#include <queue>
+#include <string>
+#include <vector>
+
+#include "ackermann_msgs/msg/ackermann_drive.hpp"
+#include "controller_interface/controller_interface.hpp"
+#include "geometry_msgs/msg/twist.hpp"
+#include "geometry_msgs/msg/twist_stamped.hpp"
+#include "hardware_interface/handle.hpp"
+#include "nav_msgs/msg/odometry.hpp"
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_lifecycle/state.hpp"
+#include "realtime_tools/realtime_box.h"
+#include "realtime_tools/realtime_buffer.h"
+#include "realtime_tools/realtime_publisher.h"
+#include "tf2_msgs/msg/tf_message.hpp"
+#include "tricycle_controller/odometry.hpp"
+#include "tricycle_controller/speed_limiter.hpp"
+#include "tricycle_controller/visibility_control.h"
+
+namespace tricycle_controller
+{
+using CallbackReturn = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn;
```

Comment:
needs to be changed to controller_interface::CallbackReturn but I'm getting this error:

`error: ‘CallbackReturn’ in namespace ‘controller_interface’ does not name a type.

`

Reply:
Actually I cloned ros2_control and ros2_controllers to master and when I try to build diff_drive_controller I get the same error. 
The rest of my system is on galactic, could it be related...